### PR TITLE
Login and Sign Up page design

### LINKED ISSUE #44: Signup with SSO
Add the signup with SSO link to the ```/signup``` endpoint.

### LINKED ISSUE #46: design login and signup page
Login and signup page choose a background color that fits TU Graz theme, and for signup page design  see [ref](https://zenodo.org/signup/)

### PR DESCRIPTION
**Pull request by freelion93**
The design of the sign-in/-up pages with regards to the semantic UI 
This closes #46, and closes #44

### CODE REVIEW

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html [hunk 1]
```diff
@@ -10,65 +10,67 @@
...
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
+
 {% block page_body %}
-  <div class="ui padded segments big form">
-    <div class="ui segment padded">
-      <div class="divider hidden"></div>
+<div class="ui centered stackable grid">
+  <div class="centered row">
+  <h2>Welcome back!</h2>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html:17]: strings must be marked for translation. see #11

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html [hunk 2]
```diff
@@ -10,65 +10,67 @@
...
-        <div class="hidden content">
-          <img src="{{ url_for('static', filename=config.INVENIO_THEME_TUGRAZ_ICON)}}" height="20px">
-        </div>
-      </div> 
-    </a>
-      
-      <div class="ui horizontal divider">
-        Or
-      </div>
+      {%- if config.INVENIO_CONFIG_TUGRAZ_SHIBBOLETH == 'True' %}
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html:27]: ```{%- if config.INVENIO_CONFIG_TUGRAZ_SHIBBOLETH %}```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 3]
```diff
@@ -10,46 +10,61 @@
...
+  <div class="ui divider"></div>
+  <div class="ui bottom attached segment sign-form">
+    <div class="ui two column stackable grid">
+      <div class="column">
+        <h4 class="ui inverted header">Citeable. Discoverable.</h4>
+        <h5 class="ui inverted header">Uploads get a Digital Object Identifier (DOI) to make them easily and uniquely citeable.</h5>
+        <h4 class="ui inverted header">Communities</h4>
+        <h5 class="ui inverted header">Accept or reject uploads to your own community (e.g workshops, EU projects, institutions or entire disciplines).</h5>
+        <h4 class="ui inverted header">Trusted Research Data Management</h4>
+        <h5 class="ui inverted header">Built on top of CERN's expertise in managing 100s of petabytes of research data from the Large Hadron Collider.</h5>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:26]: Does it say ```CERN```?

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 4]
```diff
@@ -6,50 +6,65 @@
...
-
+<div class="ui centered stackable grid">
+  <div class="centered row">
+  <h2>{{ _('Create an Account') }}</h2>
+  </div>
+  <div class="ui divider"></div>
+  <div class="ui bottom attached segment sign-form">
+    <div class="ui two column stackable grid">
+      <div class="column">
+        <h4 class="ui inverted header">Citeable. Discoverable.</h4>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:21]: Aren't these strings that user sees in the UI?
```suggestion
        <h4 class="ui inverted header">{{ _( Citeable. Discoverable.) }}</h4>
```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 5]
```diff
@@ -6,50 +6,65 @@
...
+        <h5 class="ui inverted header">Uploads get a Digital Object Identifier (DOI) to make them easily and uniquely citeable.</h5>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:22]:
```suggestion
        <h5 class="ui inverted header">{{ _(Uploads get a Digital Object Identifier (DOI) to make them easily and uniquely citeable.) }}</h5>
```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 6]
```diff
@@ -6,50 +6,65 @@
...
+        <h4 class="ui inverted header">Communities</h4>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:23]:
```suggestion
        <h4 class="ui inverted header">{{ _(Communities) }}</h4>
```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 7]
```diff
@@ -6,50 +6,65 @@
...
+        <h5 class="ui inverted header">Accept or reject uploads to your own community (e.g workshops, EU projects, institutions or entire disciplines).</h5>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:24]:
```suggestion
        <h5 class="ui inverted header">{{ _(Accept or reject uploads to your own community (e.g workshops, EU projects, institutions or entire disciplines).) }}</h5>
```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 8]
```diff
@@ -6,50 +6,65 @@
...
+        <h4 class="ui inverted header">Trusted Research Data Management</h4>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:25]:
```suggestion
        <h4 class="ui inverted header">{{ _(Trusted Research Data Management) }}</h4>
```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 9]
```diff
@@ -6,50 +6,65 @@
...
+        <h5 class="ui inverted header">Built on top of CERN's expertise in managing 100s of petabytes of research data from the Large Hadron Collider.</h5>
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:26]:
```suggestion
        <h5 class="ui inverted header">{{ _(Built on top of CERN's expertise in managing 100s of petabytes of research data from the Large Hadron Collider.) }}</h5>
```

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html [hunk 10]
```diff
@@ -6,50 +6,65 @@
...
       {%- block form_header %}
-        <h3 class="ui header">
+        <h3 class="ui inverted header">
           {% trans sitename=config.ACCOUNTS_SITENAME %}Sign up for an {{ sitename }} account!{% endtrans %}
         </h3>
       {%- endblock form_header %}
-
       {%- with form = register_user_form %}
-        <form action="{{ url_for_security('register') }}" method="POST"
-              name="register_user_form">
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:26]: what was wrong here?

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html (hunk 4 above)
Comment (freelion93) [reply invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/register_user.html:21]: This is the dummy text right now, which can be changed later,

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html [hunk 11]
```diff
@@ -1,74 +1,76 @@
...
-        <div class="hidden content">
-          <img src="{{ url_for('static', filename=config.INVENIO_THEME_TUGRAZ_ICON)}}" height="20px">
-        </div>
-      </div> 
-    </a>
-      
-      <div class="ui horizontal divider">
-        Or
-      </div>
+      <button class="ui icon inverted button" href="{{ url_for('sso_saml.sso', idp='idp') }}">
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html:28]:
```suggestion
      <a href="{{ url_for('sso_saml.sso', idp='idp') }}">
      <button class="ui icon inverted button" >
      </a/
```
The hyperlink is not working, while the ```href``` is inside a button.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html [hunk 12]
```diff
@@ -1,74 +1,76 @@
...
-{%- extends "invenio_theme/page_cover.html" %}
+{%- extends config.ACCOUNTS_COVER_TEMPLATE %}	
 {% from "invenio_accounts/_macros.html" import render_field, form_errors %}
 
+
 {% block page_body %}
-  <div class="ui padded segments big form">
-    <div class="ui segment padded">
-      <div class="divider hidden"></div>
+<div class="ui centered stackable grid">
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/login.html:15]:
```suggestion
<div class="ui container">
<div class="ui centered stackable grid">
...
```
Add the div inside a container, we need some padding at bottom see screenshot.
have a look

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/index.html [hunk 13]
```diff
@@ -127,4 +127,3 @@ <h2 class="ui center aligned icon header">
 
 </div>
 {%- endblock %}
-
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/index.html:130]: A new line is needed on this file, to resolve the conflict.

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less [hunk 14]
```diff
@@ -47,5 +54,5 @@ To #000000
 forcing the font changes
 */
 body {
-  font-family: "Source Sans Pro";
+  font-family: Source Sans Pro !important;
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/overrides.less:57]:
```suggestion
  font-family: "Source Sans Pro";
```
This change is not necessary anymore, as I mentioned in the call it took more than a body tag to fix it see #69.
I want you to revert all the changes from this file to resolve the conflict.

File: invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/theme.less [hunk 15]
```diff
@@ -9,6 +9,7 @@
 @import "header";
 @import "footer";
 @import "overrides";
+@import "accounts";
```
Comment (mb-wali) [inline invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/theme.less:12]: This file has conflict due to the current changes in the master branch, Pull the changes from master first.

File: invenio_theme_tugraz/config.py [hunk 16]
```diff
@@ -49,6 +53,9 @@
 # Invenio-accounts
 # ============
 # See https://invenio-accounts.readthedocs.io/en/latest/configuration.html
+COVER_TEMPLATE = 'invenio_theme_tugraz/accounts/accounts_base.html'
```
Comment (mb-wali) [inline invenio_theme_tugraz/config.py:56]: This file has conflict due to the current changes in the master branch, pull the changes from master first.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/accounts_base.html [hunk 17]
```diff
@@ -0,0 +1,73 @@
...
+{{ webpack['invenio-theme-tugraz-theme.css'] }}
+
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2017-2018 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
```
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/accounts/accounts_base.html:19]:
```suggestion

```

Remove these lines.

File: invenio_theme_tugraz/templates/invenio_theme_tugraz/index.html (hunk 13 above)
Comment (mb-wali) [inline invenio_theme_tugraz/templates/invenio_theme_tugraz/index.html:130]: Revert changes from this file to resolve the conflict.

File: invenio_theme_tugraz/config.py [hunk 18]
```diff
@@ -25,6 +25,9 @@
 THEME_LOGO = 'images/tug_logo.png'
 """TU Graz logo"""
 
+ACCOUNTS_HEADER_LOGO =  'images/login_logo.png'
```
Comment (mb-wali) [inline invenio_theme_tugraz/config.py:28]:
```suggestion
ACCOUNTS_HEADER_LOGO = 'images/login_logo.png'
```
The extra space is failing the travis-CI

**Pull request merged**